tests/trainer/loops/test_all.py

```diff
@@ -0,0 +1,93 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
do you prefer one catch-all test? or dedicated test files for each loop?

Reply:
no strong preference, just following the pattern we have in other places too.

Reply:
IMO this is okay 👍